File: bot/commands/util/invite.js
exports.run = async (bot) => {
  bot.registerCommand("invite", async (message, args) => {
    await bot.checkUserAndGuild(message)

    const inviteEmbed = {
      embed: {
        color: bot.getEmbedColor(bot, message),
        description: "[Click here to add Atlantis to your server!](https://discordapp.com/oauth2/authorize?client_id=688911718788628496&permissions=321600&scope=bot)"
      }
    }
    bot.createMessage(message.channel.id, inviteEmbed)
  }, {
    cooldown: 3000,
    cooldownMessage: "Whoa there slow down, the cooldown is 3 seconds!"
  })
}

File: src/main/java/dastrangeboi/atlantis/Commands/Invite.java
package dastrangeboi.atlantis.Commands;

import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;

public class Invite {
    public static void run(GuildMessageReceivedEvent message) {
        EmbedBuilder inviteEmbed = new EmbedBuilder()
                .setColor(message.getGuild().getSelfMember().getColor())
                .setDescription("[Click here to add Atlantis to your server!](https://discord.com/oauth2/authorize?client_id=688911718788628496&permissions=321600&scope=bot)");

        message.getChannel().sendMessage(inviteEmbed.build()).queue();
    }
}
